breathecode/certificate/migrations/0018_remove_specialty_syllabus_specialty_syllabus.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 5.1.7 on 2025-03-24 21:43
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("admissions", "0068_merge_20241216_1552"),
+        ("certificate", "0017_layoutdesign_foot_note"),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name="specialty",
+            name="syllabus",
+        ),
```

Comment:
Instead of removing the field, we should create a new field named syllabus where we are going to store the ManyToMany field, and then in a different PR in the future when everything is working fine, we can remove the old field